### DISCUSSION TITLE
Momentum vocoder?

### DISCUSSION THREAD

**Discussion by patty886** [ElectroTechnique/Momentum#15] [Suggestions]
Hi, would it be possible to add an audio input and code block to allow Momentum to be used as a vocoder (with the synth engine providing the carrier signal)? 
I've seen links on the PJRC site to a teensy-powered guitar multi-fx unit [https://www.pjrc.com/diy-vocoder/](url) , would adding a Teensy audio adaptor board allow this?
Thanks, Pat

**Comment by Electrotechnique** [comment]
Hello Pat,

Yes, you need to swap the PCM5102 for the Teensy Audio Adapter, but also add a connection for DOUT on the shield to a suitable pin on the MicroMod connector. Then add the input code to the audio patching file.

Regards,

Simon
Electrotechnique

________________________________
From: patty886 ***@***.***>
Sent: Thursday, December 12, 2024 5:38 PM
To: ElectroTechnique/Momentum ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [ElectroTechnique/Momentum] Momentum vocoder? (Discussion #15)


Hi, would it be possible to add an audio input and code block to allow Momentum to be used as a vocoder (with the synth engine providing the carrier signal)?
I've seen links on the PJRC site to a teensy-powered guitar multi-fx unit [url]https://www.pjrc.com/diy-vocoder/ , would adding a Teensy audio adaptor board allow this?
Thanks, Pat

**Comment by patty886** [comment]
Fantastic, thanks so much Simon.

I’ll order an Audio Adapter and start experimenting. I might need play with a different front panel design or enclosure to accommodate the audio adapter board and input jacks, it’ll probably take a little while to get to but I’ll keep you posted on progress 🙂

Cheers,
Pat

On 13 Dec 2024, at 19:09, Electrotechnique ***@***.***> wrote:

﻿

Hello Pat,

Yes, you need to swap the PCM5102 for the Teensy Audio Adapter, but also add a connection for DOUT on the shield to a suitable pin on the MicroMod connector. Then add the input code to the audio patching file.

Regards,

Simon
Electrotechnique

________________________________
From: patty886 ***@***.***>
Sent: Thursday, December 12, 2024 5:38 PM
To: ElectroTechnique/Momentum ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [ElectroTechnique/Momentum] Momentum vocoder? (Discussion #15)


Hi, would it be possible to add an audio input and code block to allow Momentum to be used as a vocoder (with the synth engine providing the carrier signal)?
I've seen links on the PJRC site to a teensy-powered guitar multi-fx unit [url]https://www.pjrc.com/diy-vocoder/ , would adding a Teensy audio adaptor board allow this?
Thanks, Pat

**Comment by Electrotechnique** [comment]
I've just realised that the schematic for Momentum wasn't on Github, so I've added it.

https://github.com/ElectroTechnique/Momentum/tree/main/Manual
[https://opengraph.githubassets.com/acfb52d14302e9592ecda071e05907423c16ebd4453837f55d00c57ba6e05545/ElectroTechnique/Momentum]<https://github.com/ElectroTechnique/Momentum/tree/main/Manual>
Momentum/Manual at main · ElectroTechnique/Momentum<https://github.com/ElectroTechnique/Momentum/tree/main/Manual>
Momentum Synth. Contribute to ElectroTechnique/Momentum development by creating an account on GitHub.
github.com


Regards,

Simon

________________________________

**Comment by patty886** [comment]
Oh perfect, thank you.
That’ll be great, I’ve also been thinking of working on a knob-per-function add on/breakout box so the schematic will really help.
So far the simplest option I can think of is to build a Tsynth without the audio board to use the mux chips, and the Teensy can handle MIDI duties to pass data to Momentum. There’s probably a more elegant way though 🙂
Cheers
Pat

On 14 Dec 2024, at 13:53, Electrotechnique ***@***.***> wrote:

﻿

I've just realised that the schematic for Momentum wasn't on Github, so I've added it.

https://github.com/ElectroTechnique/Momentum/tree/main/Manual
[https://opengraph.githubassets.com/acfb52d14302e9592ecda071e05907423c16ebd4453837f55d00c57ba6e05545/ElectroTechnique/Momentum]<https://github.com/ElectroTechnique/Momentum/tree/main/Manual>
Momentum/Manual at main · ElectroTechnique/Momentum<https://github.com/ElectroTechnique/Momentum/tree/main/Manual>
Momentum Synth. Contribute to ElectroTechnique/Momentum development by creating an account on GitHub.
github.com


Regards,

Simon

________________________________
